flashprog.passthru.tests on x86_64-darwin failed to build:

```
cli_common.c:68:86: error: declaration shadows a variable in the global scope [-Werror,-Wshadow]
int cli_parse_log_args(struct log_args *const args, const int opt, const char *const optarg)
```

--- a/pkgs/by-name/fl/flashprog/package.nix
+++ b/pkgs/by-name/fl/flashprog/package.nix
@@ -14,12 +14,12 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "flashprog";
-  version = "1.2";
+  version = "1.3";
 
   src = fetchgit {
     url = "https://review.sourcearcade.org/flashprog";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-Z09hZ4a/G3DhWCmSkPyKs7ecSFBUfez7IWWxIhH3LyI=";
+    hash = "sha256-S+UKDtpKYenwm+zR+Bg8HHxb2Jr7mFHAVCZdZTqCyRQ=";
   };
 
   nativeBuildInputs = [

